tests/path/test-path.c

```diff
@@ -0,0 +1,75 @@
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "utils.h"
+
+void compare(char *input, char *expected_output)
+{
+    char *input_sanitized = sanitize_path(input);
+    if (strcmp(input_sanitized, expected_output)) {
+        printf("\n\nInput =\t\t\t%s\nOutput =\t\t%s\nExpected output =\t%s\n",
+               input, input_sanitized, expected_output);
+
+        exit(1);
+    }
+
+    free(input_sanitized);
+}
+
+void sanitize_path_test()
+{
+    // Already clean
```

Comment:
C-style comments!

Reply:
I think I will need some more days to get my brain used to not use `//` :'(